script/sync

```diff
@@ -134,6 +134,30 @@ def sync_ruby(version)
   puts "done."
 end
 
+def sync_node(version)
+  s3_key = "nodes/Darwin/#{s3_cellar}#{os}/#{version}.tar.bz2"
```

Comment:
Use `otool -L` on the files inside `/opt/nodes/`  and see if any of them reference any paths in `/opt/boxen/homebrew`